.gitignore

```diff
@@ -1,3 +1,4 @@
 build/**
```

Comment:
Shall we delete build at the same time - or comment that it is no longer in use?

Reply:
I'm wondering why we had just the contents of `build` before - is it something to do with needing when we checkout to `gh-pages` or does it seem to be working as it is with ignoring the whole `dist` dir? Is there a reason to go with `dist` rather than keep `build`?

Reply:
`dist` is the default production dir that Webpack creates if we do not specify any other name. that will be the be the next step and it will be called `build`.